webapp/channels/src/components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -127,8 +261,10 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
         }
     };
 
-    private handleDelete = (values: UserProfileValue[]): void => {
-        this.setState({values});
+    private handleDelete = (values: Array<UserProfileValue | GroupValue>): void => {
+        // Our values for this component are always UserProfileValue
```

Comment:
Maybe we should canister keeping the groups and profiles separately? That way we don't need comments like this. Or maybe a data type for the "values" that is just for them?

Reply:
I could explore making the `values` type for the multiselect have it's own type instead of matching the `options` but not sure what react-select will think of that. I'll let you know